src/components/Search.js

```diff
@@ -112,6 +118,11 @@ class Search extends React.Component {
     render() {
         return (
             <form className={'searchForm ' + this.props.activeClassName} action="" ref={this.searchForm} onSubmit={this.fetchData}>
+                <Autocomplete
+                    onSelect={this.handleOnSelect.bind(this)}
```

Comment:
These 2 binds can be removed in favor of the class-property arrow syntax mentioned earlier.